app/lib/actions/katello/repository/sync.rb

```diff
@@ -56,6 +57,7 @@ def plan(repo, options = {})
               end
               plan_self(:id => repo.id, :sync_result => output, :skip_metadata_check => skip_metadata_check, :validate_contents => validate_contents,
                         :contents_changed => output[:contents_changed])
+              update_rolling_content_views(repo)
```

Comment:
Was having the rolling CV update be asynchronous considered? I suppose it's likely that users won't have many rolling content views, so the sync/upload shouldn't likely be blocked too long, but I'm curious if it was a consideration.